README.md

```diff
@@ -1,59 +1,61 @@
-# teracy-dev-core
+# Teracy-dev-core
 
-The core extension for teracy-dev which support all options from https://www.vagrantup.com/docs/
+The core extension for [teracy-dev](https://github.com/teracyhq/dev/tree/develop) which support all options from [vagrant](https://www.vagrantup.com/docs/)
 for teracy-dev yaml config format.
 
+## Prerequisites
+
+- Follow the guide at https://github.com/teracyhq/dev/blob/develop/docs/getting_started.rst to set up `teracy-dev`.
 
 ## How to use
 
 By default, teracy-dev already configures to use this extension, however, you can override the
 config with your own, for example, to use a different version or a forked repo.
 
-To override the built-in config, create `workspace/teracy-dev-entry/config_default.yaml` with the
+To override the built-in config, create `workspace/teracy-dev-entry/config_override.yaml` with the
 following similar content:
 
-
-- Use specific version:
+- Use latest stable version (auto update):
```

Comment:
I don't see the reason to change order here, all the README has the same order, so don't make them inconsistent.
(https://github.com/teracyhq-incubator/teracy-dev-essential#how-to-use https://github.com/teracyhq-incubator/teracy-dev-certs#how-to-use)